docs/guides/modules/test/pages/smarter-testing.adoc

```diff
@@ -41,565 +41,1571 @@ TIP: In codebases with sparse test coverage, Smarter Testing cannot accurately d
 ----
+====
 
-*Examples of `run` commands*
+Run the test suite and confirm that the `discover` command found the test atoms you expect:
+[source,console]
+----
+$ circleci run testsuite "ci tests" --local
+Running test-suite-subcommand version "1.0.14935-630104a" built "2025-11-25T16:15:39Z"
+Testsuite timeout: 4h40m0s
+Running test suite 'ci tests'
 
-[.table-scroll]
---
-[cols=2*, options="header"]
-|===
-| Test Framework| Command
+Suite Configuration:
 
-|*Jest*
-|`JEST_JUNIT_OUTPUT_FILE="<< outputs.junit >>" jest --runInBand --reporters=jest-junit --bail << test.atoms >>`
+name: ci tests
+discover:
+    command: vitest list --filesOnly
+    shell: /bin/sh
+run:
+    command: |-
+        echo 'running: <<test.atoms>>' | tr ' ' '
+        '
+    shell: /bin/sh
 
-|*Yarn Jest*
-|`JEST_JUNIT_OUTPUT_FILE="<< outputs.junit >>" yarn test --runInBand --reporters=jest-junit --bail << test.atoms >>`
 
-|*Vitest*
-|`vitest run --reporter=junit --outputFile="<< outputs.junit >>" --bail << test.atoms >>`
+Discovering...
+Discovered 2 tests in 29ms
 
-|*pytest*
-|`pytest --disable-pytest-warnings --no-header --quiet --tb=short --junit-xml="<< outputs.junit >>" << test.atoms >>`
+Selecting tests...
+Selecting all tests, no impact analysis available
+Selected 2 tests, Skipped 0 tests in 0s
 
-|*Go*
-|`go test -race -count=1 << test.atoms >>`
+Timing data is not present.
+Sorted tests in 0s
 
-|*gotestsum*
-|`go tool gotestsum --junitfile="<< outputs.junit >>" -- -race -count=1 << test.atoms >>`
-|===
---
+Running 2 tests
+   echo 'running: src/pages/dashboard/Dashboard.test.tsx src/pages/dashboard/CreateProjectButton.test.tsx' | tr ' ' '\n'
+running:
+src/pages/dashboard/Dashboard.test.tsx
+src/pages/dashboard/CreateProjectButton.test.tsx
+Ran 2 tests in 34ms
 
-=== 1.5 Update your CircleCI configuration to use the test suite
+Analysis not configured
+Not updating test impact data, analysis not enabled
+----
 
-Your `.circleci/test-suites.yml` file is now set up to match your existing way of running tests. You now need to update your CircleCI configuration to use your `test-suites.yml` file.
+=== 1.3 Configure the run command
 
-. Update the your CircleCI configuration file (`.circleci/config.yml`) to use the `circleci run testsuite "ci tests"` command.
-. Push a change and observe the step output of the test job.
+The `run` command executes the test atoms discovered by the `discover` command using a test runner. The `run` command is the same command you use to launch your test runner locally.
 
-[source,yaml]
+Make sure that the command stores JUnit results, and that it is passed a list of test atoms to run, for example:
+[source,console]
 ----
-version: 2.1
-jobs:
-  test:
-    executor: node-with-service
-    steps:
-      - setup
-      - run: circleci run testsuite "ci tests"
-      - store_test_results:
-          path: test-reports
+$ vitest run --reporter=junit \
+             --outputFile="test-reports/tests.xml" \
+             --bail 0 \
```

Comment:
Do we need the `--bail 0` in the example? We probably should add some explanation on what it's doing